docs/installation.rst

```diff
@@ -104,6 +104,10 @@ decreasing it, or speed up exports by increasing it.
 Can be overridden on a ``Resource`` class by setting the ``chunk_size`` class
 attribute.
 
+``PROCESS_WITHOUT_SHOW_CONFIRM_FORM``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It's a boolean parameter. If that parameter is true, the form to confirm 
+data inside the file to import won´t be shown and the file will be imported.
```

Comment:
**nitpick:** Looking at the [previous examples](https://github.com/juntossomosmais/django-import-export/pull/1/files#diff-d9b149498982c0663c3b7170398773361ed5678f1a627e9c2fd8d2c955c563dbR87-R102), I believe there's a missing black line between the `~~~` and the text.

```suggestion
``PROCESS_WITHOUT_SHOW_CONFIRM_FORM``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

It's a boolean parameter. If that parameter is true, the form to confirm 
data inside the file to import won´t be shown and the file will be imported.
```